packages/ui/src/routes/+page.svelte

```diff
@@ -33,7 +79,7 @@
 			<WalletConnect />
 		{/if}
 
-		<Masonry gridGap="0" colWidth={'minmax(Min(320px, 100%), 1fr)'} items={$posts.posts}>
+		<Masonry gridGap="0" colWidth={newColWidth} items={$posts.posts}>
```

Comment:
To consider: we could just call the function here instead of having a variable. But I don't really have a prefferrence here.